src/Client.php

```diff
@@ -69,6 +69,52 @@ public function get($uri)
         return $parsedResponse;
     }
 
+    public function postFile($realFilePath, $uploadFileName = null)
+    {
+        if (!$uploadFileName) {
+            // No specified filename, use the uploaded one
+            $uploadFileName = pathinfo($realFilePath)['basename'];
+        }
+        $me = $this->get('me');
```

Comment:
Actually, let's keep this internal API call. But we should save it as a protected variable and reuse this variable to not call the API each time we call postFile().
This protected variable should have a time to live equal to the TTL of signed S3 form